app.js

```diff
@@ -363,95 +362,189 @@ d3.select('#clear')
+          })
+          .attr('class', ' txt-s small py1')
+
+
+        var bar = dataRow
+          .append('div')
+          .style('width', '70%')
+          .style('display', 'inline-block')
+          .style('position', 'relative')
+          .style('float', 'right')
+          .style('height', '100%')
+
+        bar
+          .append('span')
+          .attr('class', 'z5 absolute py1 px6 txt-s')
+          .text(function(d){
+            return d+'%'
+          })
+          .each(function(d,i){
+            var color = i === maxIndex ? 'color-white' : 'color-black'
+            d3.select(this).classed(color, true)
+          })
+
+        bar
+          .append('div')
+          // .attr('class', 'round')
+          .style('width', function(d){
+            return 100 * d/data[maxIndex] + '%'
+          })
+          .style('background-color', '#448ee4')
+          .style('height', '100%')
+          .each(function(d,i){
+            var color = i === maxIndex ? 'bg-blue-light' : 'bg-darken10'
+            d3.select(this).classed(color, true)
+          })
+
+      }
+
+    // if (document.getElementById('corridor-info-table').rows.length > item) {
+    //   var row = document.getElementById(corridorInfo[item].id);
+    //   var desc = row.cells[0];
+    //   var more = row.cells[1];
+    // } else {
+    //   var row = document.getElementById('corridor-info-table').insertRow(-1);
+    //   row.id = corridorInfo[item].id;
+    //   //console.log("Added row in corridor info table: " + row.id);
+    //   var desc = row.insertCell(0);
+    //   var more = row.insertCell(1);
+    // };
+
+
+    // if (data.length == 1) desc.innerHTML =  corridorInfo[item].text + ': ' + '<span class="txt-bold">' + data[0] + '</span> ';
+    // else {
+
+
+
+    //   // // TODO: Add additional rows for the sub-items
+    //   // desc.innerHTML =  corridorInfo[item].text + '</br>';
+    //   // for (j in corridorInfo[item].secondaryText) {
+    //   //  desc.innerHTML += '<p class=secondaryText>' + corridorInfo[item].secondaryText[j] + ': ' + '<span class="txt-bold">' + data[j] + '%</span></p>';
+    //   //  //var secondaryText = document.createElement('p');
+    //   //  //secondaryText.innerHTML = '<p>' + corridorInfo[item].secondaryText[j] + ': ' + '<span class="txt-bold">' + data[j] + '</span></p>';
+    //   //  //desc.appendChild(secondaryText);
+    //   // }
+    // };
      // "More" section is a placeholder for accessing:
      // 1. Information about the data the statistic was derived from
      // (we could also consolidate all the data info in one place elsewhere)
      // 2. Toggling on/off visualizations on the map for that statistic
-     more.innerHTML = '<svg class="icon inline"><use xlink:href="#icon-question"/></svg> <svg class="icon inline"><use xlink:href="#icon-map"/></svg>'
+     // more.innerHTML = '<svg class="icon inline"><use xlink:href="#icon-question"/></svg> <svg class="icon inline"><use xlink:href="#icon-map"/></svg>'
```

Comment:
Do we still want to use this UI for for toggling on/off the display of the data attribution? If so, should we add it back now?